dask_ml/datasets.py

```diff
@@ -430,6 +431,13 @@ def make_classification_df(
         The output values.
 
     """
+    if (
+        random_state is not None
+        or not isinstance(random_state, np.random.RandomState)
+        or not isinstance(random_state, int)
+    ):
+        random_state = None
```

Comment:
Why is the block necessary? `None` is already the default value for `random_state`.

Reply:
This was to address the issue you mentioned earlier, "what if random_state is not an integer or any of the accepted values"